addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -943,6 +943,10 @@ var SnippetsMenu = Widget.extend({
             $(range && range.sc).closest('.o_default_snippet_text').removeClass('o_default_snippet_text');
         });
 
+        if ($('.o_we_snippet_autofocus').length) {
+            this._activateSnippet($('.o_we_snippet_autofocus').first());
```

Comment:
If we can search the DOM only one it's better ;)

Reply:
Done